pkgs/by-name/on/one-click-backup/package.nix

```
@@ -0,0 +1,47 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+, cmake
+, ninja
+, qt6
+#, wrapQtAppsHook
+, extra-cmake-modules
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "one-click-backup";
+  version = "1.2.2.1";
+
+  src = fetchFromGitLab {
+    owner = "dev-nis";
+    repo = "nis-one-click-backup-qt";
+    rev = finalAttrs.version;
+    hash = "sha256-JbgmfKEpKRj/ph8dkW0IRulmAhk6Dwv/1Z4j3hUgPXk=";
+  };
+
+ # nativeBuildInputs = [
+ #   cmake
+ #   ninja
+ #   wrapQtAppsHook
+ #   extra-cmake-modules
+ # ];
```

I meant just `extra-cmake-modules`, not everything in `nativeBuildInputs`.

```suggestion
 nativeBuildInputs = [
   cmake
   ninja
   qt6.wrapQtAppsHook
 ];
```

---

Oh, I misunderstood that, sorry